certified-connectors/Converter by Power2Apps/apiDefinition.swagger.json

```diff
@@ -3,676 +3,2194 @@
+        ],
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "in": "header",
+            "name": "fileName",
+            "type": "string",
+            "x-ms-visibility": "important",
+            "x-ms-summary": "File Name",
+            "description": "File name"
+          },
+          {
+            "in": "header",
+            "name": "fileExtension",
+            "type": "string",
+            "default": "Extract from File name",
+            "x-ms-dynamic-values": {
+              "operationId": "GetFileExtensions",
+              "value-path": "name",
+              "value-title": "name",
+              "parameters": {}
+            },
+            "x-ms-visibility": "important",
+            "x-ms-summary": "File Extension",
+            "description": "File extension"
+          },
+          {
+            "in": "body",
+            "name": "fileData",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/FileData"
+            },
+            "x-ms-visibility": "important",
+            "x-ms-summary": "File",
+            "description": "File"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/FileContentResponse"
+            }
+          }
+        },
+        "x-ms-visibility": "internal",
+        "summary": "Convert file (DOCX, PPTX, XLSX, HTML, ...) to PDF (deprecated)",
+        "description": "Convert a document(Word, Powerpoint, Excel, ..) to a PDF file"
+      }
+    },
+    "/ConvertFileToPdfV2": {
+      "post": {
+        "tags": [
+          "Converter"
+        ],
+        "operationId": "ConvertFileToPdfV2",
+        "consumes": [
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "in": "header",
+            "name": "fileName",
+            "type": "string",
+            "x-ms-visibility": "important",
+            "x-ms-summary": "File Name",
+            "description": "File name"
+          },
+          {
+            "in": "header",
+            "name": "fileExtension",
+            "type": "string",
+            "default": "Extract from File name",
+            "x-ms-dynamic-values": {
+              "operationId": "GetFileExtensions",
+              "value-path": "name",
+              "value-title": "name",
+              "parameters": {}
+            },
+            "x-ms-visibility": "important",
+            "x-ms-summary": "File Extension",
+            "description": "File extension"
+          },
+          {
+            "in": "body",
+            "name": "file",
+            "required": true,
```

Comment:
@stsiek The required input field has been added. Existing flows will not send this new field.

Reply:
@stsiek Please follow the same deprecation process. deprecate the current action without any changes, then add a new action as internal with new id.